### DISCUSSION TITLE
How to implement blockchain for data storage

### DISCUSSION THREAD

**Discussion by trazabest** [Cyfrin/foundry-full-course-cu#1612] [Q&A]
I wrote smart contract in solidity language on ethereum blockchain.  Front end of my  mobile app is React Native and back end of my mobile app is nodejs. But How will I implement my smart contract to interact with my mobile app and How will i write smart contract for only data storage purpose? Please reply.

**Comment by EngrPips** [comment]
Hello @trazabest, I haven't built a smart contract that I incorporated with a mobile yet so I am not in the know of how that would work but I would assume you would find useful resources on how to do that if you do some Google search, youtube search or maybe even ask an AI buddy.

**Comment by arpit3210** [comment]
@trazabest 

you'll need to use a library that can communicate with the Ethereum blockchain. One popular library for this purpose is Web3.js.

[Web3.js](https://web3js.readthedocs.io/en/v1.10.0/)  Documetation

also, you can use ether.js

[Ether.js](https://docs.ethers.org/v5/)  Documetation
